docs/database/schema-alternative.sql

```diff
@@ -0,0 +1,101 @@
+--
```

Comment:
@jimmidyson this is the sql corresponding to https://github.com/redhat-ipaas/ipaas-rest/issues/91#issuecomment-282102839 Could you do me a favor and generate the ERP diagram for this, too ?
Thanks !

Not all indexes have been added though (and don't know whether its fully correct, haven't deployed it yet).